docs/iso-string-ext.md

```diff
@@ -0,0 +1,89 @@
+ECMAScript Extended ISO-8601 Strings
+====================================
+
+ISO-8601 is the industry standard when it comes to serialization of dates and times in a computer.  However, there are two key use cases that are helpful for ECMAScript users that are not covered in ISO-8601:
+
+1. IANA Time Zone Names
+2. Calendar systems
+
+This document addresses addenda to the ISO-8601 spec for string serialization in the ECMAScript Temporal proposal.
+
+## IANA Time Zone Names
+
+There is precedent in other popular datetime libraries, such as Joda Time (now java.time), for appending time zone names in brackets.  We intend to fully support this syntax in ECMAScript, both input and output.  For example:
+
+```
+2007-12-03T10:15:30+01:00[Europe/Paris]
+```
+
+The ISO-8601 specification makes no mention of this syntax.  It is adopted here as a de-facto industry standard.
+
+The time zone string is written according to the rules of the time zone database (see [timezone.md](timezone.md)).  These rules are documented [here](https://htmlpreview.github.io/?https://github.com/eggert/tz/blob/master/theory.html):
+
+> Use only valid POSIX file name components (i.e., the parts of names other than '/'). Do not use the file name components '.' and '..'. Within a file name component, use only ASCII letters, '.', '-' and '_'. Do not use digits, as that might create an ambiguity with POSIX TZ strings. A file name component must not exceed 14 characters or start with '-'.
```

Comment:
Nitpick: suggest quoting the characters in backticks here, otherwise the Markdown gets messed up by the underscore.

Reply:
Done